google_compute_engine/accounts/accounts_utils.py

```diff
@@ -27,21 +27,36 @@
 from google_compute_engine import file_utils
 
 USER_REGEX = re.compile(r'\A[A-Za-z0-9._][A-Za-z0-9._-]*\Z')
+DEFAULT_USERADD_CMD = 'useradd -m -s /bin/bash -p * {user}'
+DEFAULT_USERDEL_CMD = 'userdel -r {user}'
+DEFAULT_USERMOD_CMD = 'groupadd {group}'
+DEFAULT_GROUPADD_CMD = 'groupadd {group}'
 
 
 class AccountsUtils(object):
   """System user account configuration utilities."""
 
   google_comment = '# Added by Google'
 
-  def __init__(self, logger, groups=None, remove=False):
+  def __init__(
+      self, logger, groups=None, remove=False, useradd_cmd=DEFAULT_USERADD_CMD,
+      userdel_cmd=DEFAULT_USERDEL_CMD, usermod_cmd=DEFAULT_USERMOD_CMD,
```

Comment:
Please set the variables to None by default, and have the class variables do the `useradd_cmd or DEFAULT_USERADD_CMD` logic when you set them below (lines 56-59).